resolver-dns/src/test/java/io/netty/resolver/dns/DnsNameResolverTest.java

```diff
@@ -63,86 +90,12 @@
 
     private static final InternalLogger logger = InternalLoggerFactory.getInstance(DnsNameResolver.class);
 
-    private static final List<InetSocketAddress> SERVERS = Arrays.asList(
```

Comment:
Does it make sense to keep this to be able to test against "real" servers during development? Enabled via some explicit command line flag?